diff -r -c Web/Server/Program.cs WebUpdated/Server/Program.cs
*** Web/Server/Program.cs	2019-01-03 13:10:01.315816700 +0100
--- WebUpdated/Server/Program.cs	2019-01-03 13:00:30.838226100 +0100
***************
*** 36,41 ****
--- 36,42 ----
  
          public void Configure(IApplicationBuilder app)
          {
+             app.UseResponseBuffering();
              app.UseSession();
  
              app.UsePhp(new PhpRequestOptions(scriptAssemblyName: "MyWebsite.1"));
diff -r -c Web/Server/Server.csproj WebUpdated/Server/Server.csproj
*** Web/Server/Server.csproj	2019-01-03 13:10:07.014659800 +0100
--- WebUpdated/Server/Server.csproj	2019-01-03 13:00:09.499600900 +0100
***************
*** 8,13 ****
--- 8,14 ----
    <ItemGroup>
      <PackageReference Include="Microsoft.AspNetCore.App" />
      <PackageReference Include="Peachpie.AspNetCore.Web" Version="0.9.26" />
+     <PackageReference Include="Microsoft.AspNetCore.Buffering" Version="0.2.2" />
    </ItemGroup>
  
    <ItemGroup>
